octoprint_auth_ldap/ldap.py

```diff
@@ -49,17 +51,15 @@ def search(self, ldap_filter, base=None, scope=ldap.SCOPE_SUBTREE):
         try:
             client = self.get_client()
             if client is not None:
-                # self.logger.debug("Searching LDAP, base: %s and filter: %s" % (base, ldap_filter))
+                self.logger.debug("Searching LDAP, base: %s and filter: %s" % (base, ldap_filter))
```

Comment:
I found taking this out and the changes below to be immensely useful for debugging.  Since they are at `debug` level, is there any reason to leave them commented out?

I'm also happy to split these changes into a separate commit and/or PR.

Reply:
@gillg apologies for the delay.  I pulled out this change and the related one below it out - so there's no more additional logging as part of this pull request.  I can submit those for another pull request with a proper demonstration that credentials are not leaked - at least when `$TIME` permits :)